utils/ccloud_library.sh

```diff
@@ -1240,6 +1240,7 @@ function ccloud::generate_configs() {
   ################################################################################
   KSQLDB_SERVER_DELTA=$DEST/ksqldb-server-ccloud.delta
   echo "$KSQLDB_SERVER_DELTA"
+  rm -f $KSQLDB_SERVER_DELTA
```

Comment:
IMO we do not need `-f` here.  I tested this command without, and considering the user previously created the file in a folder that they have proper permissions on, the `-f` isn't required.  So introducing the force option seems like an unecessary risk, in case something important (oddly) happen to get into the KSQLDB_SERVER_DELTA variable.

If we want to silence error messages if the file doesn't exist, i suggeste we use `rm $KSQLDB_SERVER_DELTA 2>/dev/null`

Reply:
I think that's a fair point @rspurgeon . Can I assume you'd want to alter how the other delta files are deleted in this function? 
```
rm -f $REPLICATOR_PRODUCER_DELTA
rm -f $SR_CONFIG_DELTA
rm -f $INTERCEPTORS_CONFIG_FILE
rm -f $KSQL_DATAGEN_DELTA
rm -f $C3_DELTA
rm -f $METRICS_REPORTER_DELTA
rm -f $REST_PROXY_DELTA
rm -f $CONNECT_DELTA
rm -f $CONNECTOR_DELTA
... (list goes on) ...
```
Personally I think if we change this one then we should change all of them.

Reply:
Thanks @awalther28 ... If you're up for changing all in this function, I agree.  IMO it's ok to not introduce any new usage of `-f` to start, but I don't think we should be force removing files on our users machines generally.  But starting here seems great to me